src/Symfony/Component/DomCrawler/Form.php

```diff
@@ -250,6 +250,18 @@ public function getMethod()
         return $this->node->getAttribute('method') ? strtoupper($this->node->getAttribute('method')) : 'GET';
     }
 
+    /**
+     * Gets the form name.
+     *
+     * If no name is defined on the form, an empty string is returned.
+     *
+     * @return string The name
```

Comment:
should be replaced by a real return type

Reply:
Not sure if I get your point: the method always returns string ('cause `\DOMElement::getAttribute` always returns string).

So `string` is a real return type :)

Reply:
you should add the return type like this
```php
public function getName(): string
```

Reply:
Done!

Reply:
Now that the type hint has been added on the method, the phpdoc can be removed.

Reply:
Remove the phpdoc